src/controllers/hotkeys/ActionNames.hpp

```diff
@@ -21,6 +38,19 @@ struct ActionDefinition {
 
     // maxCountArguments is the maximum amount of arguments the action accepts
     uint8_t maxCountArguments = minCountArguments;
+
+    // possibleArguments is empty or contains all possible argument values,
+    // left string from the pair is the thing the action will see,
+    // right one is what the user sees
+    std::vector<std::pair<std::vector<QString>, QString>> possibleArguments =
+        {};
```

Comment:
As discussed in chat, would be clearer if this was an `std::map<QString (what the user sees), std::vector<QString> (what arguments will be sent to the hotkey)>`

maybe vector of pairs, but the order needs to be the order they were added in at least

Reply:
I swapped the order of the pairs in [1400424](https://github.com/Chatterino/chatterino2/pull/4628/commits/1400424042cdc1ee2e8be6aa5fbaff5df6af13af)